Automattic/wp-calypso
# Better click target for domain management three-dot menu

#### Proposed Changes

* Add padding to ellipsis/three-dot menu to increase the click area

<img width="316" alt="Screen Shot 2023-01-04 at 2 14 45 pm" src="https://user-images.githubusercontent.com/1842363/210477758-f7edc11e-7e11-4202-a46f-c4f9f1dd7b7e.png">


#### Testing Instructions

* Switch to a site with a custom domain
* Go to Upgrades > Domains to see the domain list
* Test that the ellipsis/three-dot menu is 

#### Pre-merge Checklist





Related to #56230
